ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -46,22 +46,22 @@ ethernet_intefaces:
     <...>
 ```
 
-to:
+New (5.0):
```

Comment:
this is weird - line 41 states `it was changed from`... so `to` makes more sense no?

Reply:
I think this was made before we did the diff-style. Would it make sense to change it completely?

Reply:
Yeah my bad.  thinking about it "to" signals it existed before it was just updated vs a completely new thing. I really like the diff option. I can make the update